frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/ReferenceTab/ReferenceTab.test.tsx

```diff
@@ -24,14 +37,24 @@ describe('ReferenceTab', () => {
         optionsId: 'some-id',
       },
     });
-    expect(screen.getByDisplayValue('some-id')).toBeInTheDocument();
+
+    expect(getInputElement()).toHaveValue('some-id');
+  });
+
+  it('should render no value if optionsId is a codeList from the library', () => {
+    renderReferenceTab({
+      componentProps: {
+        optionsId: 'test1',
```

Comment:
Maybe `test1` not should be hardcoded but a variable that is used both in the query and here? So it is clear that it is the same?